docs/admin/integrations/platformx.md

```diff
@@ -8,21 +8,50 @@ their experience.
 
 ![PlatformX Events in Coder](../../images/integrations/platformx-screenshot.png)
 
-## Requirements
+## coder-platformx-events-middleware
 
-You'll need:
+Coder sends [notifications](../monitoring/notifications/index.md) via webhooks
+to coder-platformx-events-middleware, which processes and reformats the payload
+into a structure compatible with [PlatformX by DX](https://help.getdx.com/en/articles/7880779-getting-started).
 
-- A running Coder v2.19+ deployment
-- A PlatformX subscription from [DX](https://getdx.com/)
-- A platform to host the integration (choose any of these):
-  - AWS Lambda
-  - Heroku
-  - Kubernetes
-  - Or any other platform that can run Python web applications
```

Comment:
Where should we put the requirements?

Reply:
that was my mistake - I think I cut it while moving it and then didn't put it back

un-removed it